### PR TITLE
Fix missing Featured Image translations

### PR DESCRIPTION
**Pull request by jhnstn**
Fix missing translations on Android
See GB : https://github.com/WordPress/gutenberg/pull/37956

Partial Fix for #4432

To test:
See test step on https://github.com/WordPress/gutenberg/pull/37956

**NOTE:** This issue only affects the Android platform.
_(Edited by @fluiddot)_

PR submission checklist:

- [ ] I have considered adding unit tests where possible.
- [ ] I have considered if this change warrants user-facing release notes [more info](https://github.com/wordpress-mobile/gutenberg-mobile/blob/develop/Release-notes.md) and have added them to [RELEASE-NOTES.txt](https://github.com/wordpress-mobile/gutenberg-mobile/blob/develop/RELEASE-NOTES.txt) if necessary.

### CODE REVIEW

**Review by fluiddot — CHANGES_REQUESTED** [review]
I think the localization strings file for iOS should be also updated as the string now has a different value ([reference](https://github.com/wordpress-mobile/gutenberg-mobile/blob/develop/bundle/ios/GutenbergNativeTranslations.swift)). You could update both by running the command `npm run i18n:update`.

File: bundle/android/strings.xml [hunk 1]
```diff
@@ -142,7 +142,7 @@
     <string name="gutenberg_native_duplicate_block" tools:ignore="UnusedResources">Duplicate block</string>
     <string name="gutenberg_native_each_block_has_its_own_settings_to_find_them_tap_on_a_block_its_s" tools:ignore="UnusedResources">Each block has its own settings. To find them, tap on a block. Its settings will appear on the toolbar at the bottom of the screen.</string>
     <string name="gutenberg_native_edit_cover_media" tools:ignore="UnusedResources">Edit cover media</string>
-    <string name="gutenberg_native_edit_file" tools:ignore="UnusedResources">Edit file</string>
+    <string name="gutenberg_native_edit_file" tools:ignore="UnusedResources">Edit file </string>
```
Comment (fluiddot) [inline bundle/android/strings.xml]: Not sure if this is accurate, this string is referenced [here](https://github.com/WordPress/gutenberg/blob/1f1bccf711a9f783881ea51b3f0bf6b32c4ebc18/packages/block-library/src/file/edit.native.js#L232) but it doesn't include the extra space 🤔 .

File: bundle/android/strings.xml [hunk 2]
```diff
@@ -311,7 +311,7 @@ translators: Block name. %s: The localized block name -->
     <!-- translators: %s: Select font size option value e.g: "Selected: Large".
 translators: %s: Select control option value e.g: "Auto, 25%". -->
     <string name="gutenberg_native_selected_s" tools:ignore="UnusedResources">Selected: %s</string>
-    <string name="gutenberg_native_set_as_featured_image" tools:ignore="UnusedResources">Set as Featured Image </string>
+    <string name="gutenberg_native_set_as_featured_images" tools:ignore="UnusedResources">Set as Featured Images </string>
```
Comment (fluiddot) [inline bundle/android/strings.xml]: I couldn't find a commit where this string was changed, do you have any clue why it changed?

Comment (jhnstn) [reply bundle/android/strings.xml]: good catch, I _think_ I might have changed this and the other change above while trying to figure out the issue.

Comment (jhnstn) [reply bundle/android/strings.xml]: I know what I did 🤦 , I ran the `i18n:update` command while on the wrong gutenberg ref.

Comment (fluiddot) [reply bundle/android/strings.xml]: Oh, that explains the changes 😅.

Comment (jhnstn) [reply bundle/android/strings.xml]: Fixed in https://github.com/wordpress-mobile/gutenberg-mobile/pull/4464/commits/5821350c3c9bd9d127fe04e33fba2115c61bccdf

**Review by fluiddot — APPROVED** [review]
LGTM 🎊 !

Approved via https://github.com/WordPress/gutenberg/pull/37956#pullrequestreview-852844649.

**NOTE:** I see that the Gutenberg ref is not pointing to the latest commit of https://github.com/WordPress/gutenberg/pull/37956, we should update it once the Gutenberg PR is merged.

**Comment by fluiddot** [comment]
@jhnstn I'm wondering if we should update the PR's title in order to describe more accurately the introduced change, wdyt?
On the other hand, since the Gutenberg PR added an entry to the changelog file, probably we should do the same in the release notes.

**Comment by fluiddot** [comment]
I noticed while testing the iOS beta version `19.1 (19.1.0.0)` that the strings referenced in the [Gutenberg PR](https://github.com/WordPress/gutenberg/pull/37956) are not translated. I guess this is caused by the fact that the strings have changed and localization files in WordPress-iOS haven't been updated yet ([reference](https://github.com/wordpress-mobile/WordPress-iOS/blob/1a91e9c6ec630abd6e595fc5b3c9c01b75691d0d/WordPress/Resources/es.lproj/Localizable.strings#L6361)).

**Not translated strings:**
- `Set as Featured Image`
- `Remove as Featured Image`

**Pull request merged**